user_guide_src/source/changelogs/v4.2.0.rst

```diff
@@ -72,6 +72,7 @@ Database
 - QueryBuilder raw SQL string support
     - Added the class ``CodeIgniter\Database\RawSql`` which expresses raw SQL strings.
     - :ref:`select() <query-builder-select-rawsql>`, :ref:`where() <query-builder-where-rawsql>`, :ref:`like() <query-builder-like-rawsql>`, :ref:`join() <query-builder-join-rawsql>` accept the ``CodeIgniter\Database\RawSql`` instance.
+- QueryBuilder. Union queries.
```

Comment:
If there is a link to the detailed page, readers can go and see it easily.

Reply:
added